### PR TITLE
BUGFIX: Replace --privileged with the specific device.

### PR DESCRIPTION
**Pull request by d6e**
Using docker's privileged mode is a really bad security
practice as it grants the container full control over
the host. This resolves issue #48

### CODE REVIEW

**Comment by jneilliii** [comment]
This won't work for windows based deployments will it?

**Comment by d6e** [comment]
What devices do Windows deployments need?

**Comment by jneilliii** [comment]
I'm not sure actually and after thinking about it the image references a pi build of ffmpeg so I guess it's a moot point. I just know there have been issues posted relative to running the docker image on windows and I gave guidance on changing the entry.sh file reference to a windows build.

**Comment by jneilliii** [comment]
Would you mind testing out my changes and verify everything is working properly?  You can install using the url below in plugin manager.

https://github.com/jneilliii/OctoPrint-YouTubeLive/archive/0.5.1.zip

**Comment by d6e** [comment]
I've installed the plugin, but I can't actually test it until Youtube approves my account for live-streaming and gives me a key (they say it will take 24 hours):  https://imgur.com/pkxTPZx

Full Context:
Originally, I just wanted to stream to Twitch, so I installed the [webcam streamer plugin](https://github.com/adilinden-oss/octoprint-webcamstreamer), but then after reading through the code I noticed this issue and so I found out what devices needed to be exposed and tested that change with a fork. Then after making that PR, I noticed it was based on this repo which had the same issue, so I made this PR. 

TLDR: I can't verify until tomorrow.

**Comment by d6e** [comment]
I can confirm it works.
